static/bidder-info/adklip.yaml

```diff
@@ -0,0 +1,8 @@
+aliasOf: aso
+endpoint: "https://srv.adklip.net/pbs/bidder?zid={{.ZoneID}}"
```

Comment:
can you please provide value for ZoneID which needs to be replaced here. We need to test if endpoint is reachable.

Reply:
Verfied endpoint using zoneId value mentioned in the docs, Kindly check below. If this incorrect kindly provide valid value.

<img width="1716" height="365" alt="Image" src="https://github.com/user-attachments/assets/47153c79-cfe4-481e-82e3-540fab193ff0" />